github/Repository.py

```diff
@@ -2192,7 +2192,7 @@ def get_issues(
         :param state: string. `open`, `closed`, or `all`. If this is not set the GitHub API default behavior will be used. At the moment this is to return only open issues. This might change anytime on GitHub API side and it could be clever to explicitly specify the state value.
         :param assignee: string or :class:`github.NamedUser.NamedUser` or "none" or "*"
         :param mentioned: :class:`github.NamedUser.NamedUser`
-        :param labels: list of :class:`github.Label.Label`
+        :param labels: list (of string or :class:`github.Label.Label`)
```

Comment:
I don't like the brackets, I think just remove those and say strings, and this is fine to land.

Reply:
Have removed brackets 👍